docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -66,12 +59,6 @@ you can change:
 - `attatchment>` >> `attachment>`
 - `="games.strategy.triplea.attatchments` >> `="games.strategy.triplea.attachments`
 
-For a more defined change, instead of the aforementioned:
-
-- `Attatchment` >> `Attachment`
-
-you can change:
-
 - `="techAttatchment` >> `="techAttachment`
```

Comment:
This is now given as an alternative of "`attatchment` >> `attachment`", which is wrong, since it is, instead, an alternative of "`Attatchment` >> `Attachment`". The same problem exist for all the five following lines.

Reply:
This is an existing line @Cernelius ; can you clarify how that is "wrong" , can you give an example where following the instructions a reasonable person would make a mistake?

Reply:
Here I'm just specifying the matter I noticed above, for this and the following 5 lines. This line is an alternative change for the general "`Attatchment` >> `Attachment`", while now it is given as the same for "`attatchment` >> `attachment`".

Assuming the case matters, this is wrong, because, if I start from `="techAttatchment` and change "`attatchment` >> `attachment`" either nothing happens (if my search was case sensitive) or the code is changed to `="techattachment` (instead of `="techAttachment`) (if my search was case insensitive).

It is not the line itself that is wrong. The fact that it is given as an alternative of "`attatchment` >> `attachment`" is wrong.